### ISSUE TITLE
Show Applications appears badly broken

### ISSUE THREAD

**Issue by aryehweiss** [pop-os/pop#1081]
**Distribution (run `cat /etc/os-release`):**
 amw@pop-os:~$ cat /etc/os-release
NAME="Pop!_OS"
VERSION="20.04 LTS"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 20.04 LTS"
VERSION_ID="20.04"
HOME_URL="https://system76.com/pop"
SUPPORT_URL="http://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal
LOGO=distributor-logo-pop-os

 
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**
"Show Applications" in dock. I do not know which package it comes from

**Issue/Bug Description:**
Activating "Show Applications" results in an empty screen(the docks are still there)
The problem is that there is no way out. The applications launch, but the display remains empty. 
The workspaces show the application window has opened, but there is no way to get it to display.
So far the only way out is to log out.

Is there a magic key sequence that will get me out of there? 

**Steps to reproduce (if you know):**
Just press the "Show Application" icon in the dock. 

**Expected behavior:**
A screen full of icons to launch applications, and more important, a way to exit back to the workspaces.

**Other Notes:**
I saw this issue a while ago while searching the web, but I cannot find the post again.
I thought that I could just avoid "Show Applications", but sometimes it launches for reasons that 
I do not know (maybe I accidentally hit the wrong hot-key...).

I apologize in advance if there is a trivial solution that I missed.

**Comment by aryehweiss** [comment]
I should add that it happened after upgrading from 19.04 to 20.04

**Comment by aryehw** [comment]
Problem is now more severe, because after entering "Show Applications", POP-KEY+Esc sends me to the unlock screen window, and when I do that, it sends me back to "Show Applications". 
What this means is that once I am in "Show Applications", the only way out is to logout or restart.

**Comment by banifou** [comment]
Still no fix for this issue !?
I have the same problem.
Is there another way to see all the applications installed ?

**Comment by aryehw** [comment]
This issue was solved, at least on my system. Pressing the "windows" key on the keyboard brings me back to the working area.
I still have problems with the window manager, and have to use 
killall -3 gnome-shell 
quite often to restore sanity in the window manager.

Rendering of hidpi screens on low dpi screens (ie using an external screen with lower resolution than my 4k laptop display) is not reliable.

Once in a while the computer does not wake form suspend, and requires a hard reset.

My suggestion is not to run POP OS on hardware the System 76 does not have (like the Lenovo X1 Extreme).

**Comment by banifou** [comment]
Hmm, it's too late now. I have POP OS running on Lenovo thinkpad P1 gen2 for a year now.

**Comment by aryehw** [comment]
Yes - me too. The update to 20.04 was very problematic. My system is usable (especially since I found the killall -3 gnome-shell workaround, so that I do not have to constantly  reboot, but it is not for the faint of heart.

The firmware update problem is aggravating. I can  say that on a system that runs only  Linux it works , but if you have a non-standard UEFI setup (in my case, a Windows OS in the first 100GB), then it messes up, and you can  only boot Windows. I managed to repair the boot loader, but it is not fo rte faint of heart.

**Comment by banifou** [comment]
Sorry to hear that. In my case, it is not so bad. I'm using linux only.
I have to manage to start my apps easily, without "Show Applications".

**Comment by manishmajhi79** [comment]
This helped me!

https://askubuntu.com/questions/1230515/my-applications-are-not-showing-after-upgrading-from-ubuntu-18-04-to-20-04
